convert_hcl_to_json.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+install_jq(){
+    OS=$(uname -s)
+    if [[ "$OS" == "Darwin" ]]; then
+        OS="macos"
+    elif [[ "$OS" == "Linux" ]]; then
+        OS="linux"
+    else
+        echo "Unsupported OS: $OS"
+        exit 1
+    fi
```

Comment:
You could also pull this to a function because it's being reused in other places. Something like `_detect_os`, and call it `OS="$(_detect_os)"`